torchvision/transforms/functional.py

```diff
@@ -407,6 +407,23 @@ def hflip(img):
     return img.transpose(Image.FLIP_LEFT_RIGHT)
 
 
+def perspective(img, coeffs, interpolation=Image.BICUBIC):
+    """Perform perspective transform of the given PIL Image.
+
+    Args:
+        img (PIL Image): Image to be transformed.
+        coeffs (tuple) : 8-tuple (a, b, c, d, e, f, g, h) which contains the coefficients.
```

Comment:
those coeffs in here are very magical from an user perspective: what do they represent?
Would we want to instead provide to the user some more manageable coefficients (like the 2 quadrilaterals), and perform the 8-tuple conversion inside the function?

Reply:
Will just make the input as two quadilaterals,